src/openrct2-ui/WindowManager.cpp

```diff
@@ -32,51 +32,51 @@ class WindowManager final : public IWindowManager
     void Init() override
     {
         ThemeManagerInitialise();
-        window_new_ride_init_vars();
+        window_staff_list_init_vars();
```

Comment:
incorrect substitution.

Reply:
How did clang-tidy manage to do that?

Reply:
No idea but its very annoying that it does this.